docs/mnist_example/README.md

```diff
@@ -198,13 +198,13 @@ int main() {
 
   // Create a tensor using omTensorCreateWithOwnership (returns a pointer to OMTensor).
   // When the parameter, owning is set to "true", the OMTensor will release the data
-  // pointer upon destruction. If owning is set to false, the data pointer will not be
+  // pointer upon destruction. If owning is set to false, the tensor array will not be
   // freed upon destruction.
   OMTensor *tensor = omTensorCreateWithOwnership(img_data, shape, rank, ONNX_TYPE_FLOAT, /*owning=*/true);
 
   // Create a tensor list using omTensorListCreateWithOwnership (returns a pointer to the OMTensorList).
   // When the parameter, owning is set to "true", the OMTensorList will release the data
-  // pointer upon destruction. If owning is set to false, the data pointer will not be
+  // pointer upon destruction. If owning is set to false, the tensor array will not be
```

Comment:
A couple of notes:
1. in the same sentence, one part has "data pointer" and another has "tensor array". These should be the same.
1. I think "tensor" makes more sense than "data pointer" but it could be a little more explcit given the complexity of the mechanics. So something like...
```
// When the parameter, "owning" is set to "true", the OMTensorList will release its array of OMTensor pointers 
// upon destruction. If owning is set to "false", the array of OMTensor pointers will not be released. 
// Regardless of "owning", all of the OMTensor objects themselves are released with the OMTensorList
```

Reply:
Fixed